rasa/nlu/extractors/crf_entity_extractor.py

```diff
@@ -158,7 +158,7 @@ def _check_spacy_doc(self, message):
                 "message {}\n"
                 "POS features require a pipeline component "
                 "that provides `spacy_doc` attributes, i.e. `SpacyNLP`. "
-                "See https://nlu.rasa.com/pipeline.html#nlp-spacy "
+                "See https://rasa.com/docs/rasa/nlu/choosing-a-pipeline/#pretrained-embeddings-spacy "
```

Comment:
We should probably use the DOCS_BASE_URL env var for this

Reply:
Good idea, didn't know about that variable